File: tests/fixtures/aliases/csharp/Aliases.cs
using Proc = System.Diagnostics.Process;
using System.Diagnostics;
using static System.Diagnostics.Process;

class Aliases
{
    void Run(string command)
    {
        System.Diagnostics.Process.Start(command);
        Proc.Start(command);
        Process.Start(command);
        Start(command);
    }

    void ShadowedAlias(dynamic Proc, string command)
    {
        Proc.Start(command);
    }
}

File: tests/fixtures/aliases/typescript/aliases.ts
const cp = require("child_process");
const { exec: run } = require("child_process");

export function review(command: string) {
  cp.exec(command);
  run(command);
}

export function shadowed(cp: { exec(command: string): void }, command: string) {
  cp.exec(command);
}
